packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -127,7 +127,7 @@ BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 6.0.0
 # end package.json devDependencies BuildRequires
 # start package.json dependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(angular) = 1.8.2
-BuildRequires: %{?scl_prefix}npm(bootstrap-select) = 1.13.6
+BuildRequires: %{?scl_prefix}npm(bootstrap-select) = 1.13.18
```

Comment:
this will make this PR fail, sorry

Reply:
@evgeni should i update it here or as a 2nd pr?  I can never keep it straight :)

Reply:
Another PR, as it's a BR which needs to exist before Katello can be built